.github/workflows/deploy.yml

```diff
@@ -10,51 +10,57 @@ on:
       - galactic
       - foxy
 
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
 jobs:
-  htmlproofer:
-    name: htmlproofer
+  doc:
+    name: Build + Test
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
     - uses: actions/setup-python@v4
       with:
         python-version: '3.8'
         cache: 'pip'
+
     - name: Install Python dependencies
       run: |
         python -m pip install --upgrade pip
         pip install --upgrade --requirement requirements.txt
+
+    - name: Install doxygen and graphviz
+      run: sudo apt-get install -y doxygen graphviz
+
     - uses: ruby/setup-ruby@v1
       with:
         ruby-version: '2.7'
+
     - name: Run htmlproofer.sh
       run: ./htmlproofer.sh
 
-  deploy-multiversion:
-    needs: htmlproofer
-    if: ${{ github.event_name == 'push' }}
-    name: Deploy Multiversion
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-      with:
-        fetch-depth: 0
-    - uses: actions/setup-python@v4
-      with:
-        python-version: '3.8'
-        cache: 'pip'
-    - name: Install Python dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install --upgrade --requirement requirements.txt
-    - name: Install doxygen and graphviz
-      run: sudo apt-get install -y doxygen graphviz
     - name: Build multiversion with API
       run: make multiversion-with-api
-    - name: Deploy Pages
-      if: ${{ success() }}
-      uses: peaceiris/actions-gh-pages@v3
+
+    - name: Upload pages artifact
+      uses: actions/upload-pages-artifact@v1
+      if: github.repository_owner == 'ros-planning'
       with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        publish_dir: ./build/html
-        cname: moveit.picknik.ai
```

Comment:
don't we still have to set the cname somewhere?

Reply:
I just pushed a fix for this.